fem/fe_coll.cpp

```diff
@@ -1504,6 +1530,8 @@ const int *RT1_3DFECollection::DofOrderForOrientation(Geometry::Type GeomType,
 
 
 H1_FECollection::H1_FECollection(const int p, const int dim, const int btype)
+   : FiniteElementCollection(p)
+   , dim(dim)
```

Comment:
Not a big deal, but this is unusual style. Was it intentional, or an artifact of `make style`?

Reply:
Yeah this is a style that I brought from another project, I can remove it if there are objections.

Reply:
I am flexible on this. I'll put the comma on the previous line (or fit both on one line) if it was me.